simtools/data_model/metadata_model.py

```diff
@@ -66,9 +66,7 @@ def get_default_metadata_dict(schema_file=None, observatory="CTA"):
 
     """
 
```

Comment:
This debug statement was duplicated, as exactly the same one is printed in _load_schema